app/components/spotlight/progress_bar_component.html.erb

```diff
@@ -0,0 +1,30 @@
+<div data-behavior='progress-panel' data-exhibit-id="<%= exhibit.id %>">
+  <div class="card index-status border-info mb-3">
+    <h3 class="card-header bg-info text-white h4"><%= t("#{translation_field}.heading") %></h3>
```

Comment:
Instead of doing string interpolation with `translation_field` could you instead do `t('heading', scope: translation_field)`?